server/flyway/sql/V55__create_permission_audit_and_code_tables.sql

```diff
@@ -0,0 +1,90 @@
+-- Create a permission audit table and a code table https://github.com/bcgov/nr-forests-access-management/issues/1536
+-- Create the fam_privilege_change_type table
+CREATE TABLE app_fam.fam_privilege_change_type (
+    privilege_change_type_code VARCHAR(10) PRIMARY KEY,
+    description VARCHAR(100) NOT NULL,
+    effective_date TIMESTAMP(6) WITHOUT TIME ZONE DEFAULT NOW() NOT NULL,
+    expiry_date TIMESTAMP(6) WITHOUT TIME ZONE,
+    update_date TIMESTAMP(6) WITHOUT TIME ZONE
+);
```

Comment:
Hi Craig, could you also add `COMMENT ON COLUMN` for **fam_privilege_change_type** table columns? Thanks!

Reply:
i don't think having comments on such a simple and self-explanatory table provides any value, but i'll do it.
Does this team put comments on every table and every column?

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1577/commits/e7dba639a96ef0e6d6b22b9c683b329219a19e42